fossunited/api/emailing.py

```
@@ -541,28 +546,21 @@ def get_formatted_attachment_list(attachments: list) -> list:
 
 
 @frappe.whitelist()
+@require_mailing_access(campaign_param="campaign_id")
 def send_campaign(campaign_id: str):
```

Ruff (0.15.2)

[warning] Unused function argument: `campaign_id`

(ARG001)